### PR TITLE
This is my first Pull Request (PR)

### PR DESCRIPTION
**Pull request by ghost**
Created a txt file with some text

### CODE REVIEW

**Comment by amigoscode** [comment]
this is a comment

File: my-first-pr.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+this is my first pr
```
Comment (jamila-amigoscode) [inline my-first-pr.txt:1]: why do you have this txt file which makes no sense

Comment (amigoscode) [reply my-first-pr.txt:1]: i dont know

**Review by jamila-amigoscode — CHANGES_REQUESTED** [review]
i think these changes are pointless

**Review by jamila-amigoscode — APPROVED** [review]
LGTM

**Review by jamila-amigoscode — CHANGES_REQUESTED** [review]
opps my mistake

**Review by jamila-amigoscode — APPROVED** [review]
LGTM

**Pull request merged**